pkg/seeds/seeds.go

```diff
@@ -56,19 +57,11 @@ func SeedRepositoryConfigurations(db *gorm.DB, size int, options SeedOptions) er
 		return err
 	}
 
-	arch := "x86_64"
-	if options.Arch != nil && *options.Arch != "" {
-		arch = *options.Arch
-	}
-	versions := []string{"9"}
-	if options.Versions != nil {
-		versions = *options.Versions
-	}
 	for i := 0; i < size; i++ {
 		repoConfig := models.RepositoryConfiguration{
 			Name:           fmt.Sprintf("%s - %s - %s", RandStringBytes(2), "TestRepo", RandStringBytes(10)),
-			Versions:       versions,
-			Arch:           arch,
+			Versions:       createVersionArray(options.Versions),
+			Arch:           createArch(options.Arch),
```

Comment:
i didn't realize a previous pr had removed these methods of creating random data, so i added them back

Reply:
random orgID is gone too, not sure if that's desired